release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -937,10 +937,20 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-bare-metal-hardware-bug-fixes"]
 ==== Bare Metal Hardware Provisioning
 
+* Previously, when writing the RHCOS image onto some disks, the qemu-img was allocating space onto the entire disk, including sparse areas. This extended the time of the writing process on some hardware. This update disables the qemuimg sparse in image creation. As a result, the image writing would no longer take a long time on effected hardware. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002009[*BZ#2002009*])
+
+* Previously, if the “rotational” field was set for `RootDeviceHints`, the host could fail the provision. With this update, the properky copies and checks the “rotational” field in RootDeviceHints. As a result, the provisioning will succeed when using the “rotational” field. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2053721[*BZ#2053721*])
```

Comment:
s/the properky/the installer properly

@opayne1 Please check with the doc text author on this one. Not sure if I have the fix right. But, _properky_ is certainly wrong.

Reply:
s/"rotational"/`rotational` in three places

Reply:
s/will succeed/succeeds

Reply:
looks like it properky was a typo in the doc text in the BZ and transferred over. I have updated.